packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -92,10 +164,16 @@ export class Accelerator extends cdk.Resource implements IAccelerator {
     const resource = new ga.CfnAccelerator(this, 'Resource', {
       enabled: props.enabled ?? true,
       name,
+      ipAddresses: props.ipAddresses ?? undefined,
+      // ipAddressType is undefined as a default here for backwards compatibility,
+      // but the service selects "IPV4".
+      ipAddressType: props.ipAddressType ?? undefined,
     });
 
     this.acceleratorArn = resource.attrAcceleratorArn;
     this.dnsName = resource.attrDnsName;
+    this.ipAddresses = props.ipAddresses ?? undefined;
```

Comment:
As the use case, shouldn't we expose `Ipv4Addresses` instead of `props.ipAddresses` specified from props? (In addition, we may want to have `Ipv6Addresses` and `DualStackDnsName`, etc.)

> Ipv4Addresses
> The array of IPv4 addresses in the IP address set. An IP address set can have a maximum of two IP addresses.
> 
> Ipv6Addresses
> The array of IPv6 addresses in the IP address set. An IP address set can have a maximum of two IP addresses.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-globalaccelerator-accelerator.html#aws-resource-globalaccelerator-accelerator-return-values